docs/workflows.md

```diff
@@ -101,63 +101,83 @@ Examples:
 
 Naming Constraints
 
-Only Alphanumeric characters are allowed (A-Z+0-9, case insensitive) and no white spacesYou will need to specify the name of the output variable and the value of it. The values that are outputted can be either a free text or a token from inside the workflow execution context. (eg: `[Entity:Property]` or `[MyValue]`). Those values will be passed back where the Workflow was executed from.
+Only Alphanumeric characters are allowed (A-Z+0-9, case insensitive), and no white spaces. You will need to specify the name of the output variable and the value of it. The values that are outputted can be either a free text or a token from inside the workflow execution context. (eg: `[Entity:Property]` or `[MyValue]`). Those values will be passed back to the context where the Workflow was executed.
 
 ### `Process`
 
-#### `What are actions`
+#### `What Are Actions?`
 
-Actions are blocks of code that are packed and ready to use in a workflow. There are various actions out of the box that can interact with the stored data (Run SQL query, Read Entity), external data (Server Request), deliver messages (Send Email),business logic (Execute JavaScript (Server)) or modules themselves (Execute JavaScript and stop ,Refresh Grid)  They only need parameters/configurations to work and deliver business logic. Usually actions need some input parameters and based on them will execute their specific code. Some of the actions also have output tokens, the result of the code being executed, that can be used in the next actions. This will give you the ability to create complex flows and obtain your custom business. When the workflow ends you can attach the output tokens from the actions to the output parameters of the workflow.
+Actions are blocks of code that are packed and ready to use in a workflow. There are various actions out of the box that can interact with the stored data (Run SQL query, Read Entity), external data (Server Request), deliver messages (Send Email), business logic (Execute JavaScript (Server)), or modules themselves (Execute JavaScript and Stop Execution, Refresh Grid) They only need parameters/configurations to work and deliver business logic. Usually, actions need some input parameters based on which they will execute their specific code. Some of the actions also have output tokens, the result of the code being executed, that can be used in the next actions. This will give you the ability to create complex flows and obtain your custom business logic. When the workflow ends you can attach the output tokens from the actions to the output parameters of the workflow.
```

Comment:
```suggestion
Actions are blocks of code that are packed and ready to use in a workflow. Various actions out of the box can interact with the stored data (Run SQL query, Read Entity), external data (Server Request), deliver messages (Send Email), business logic (Execute JavaScript (Server)), or modules themselves (Execute JavaScript and Stop Execution, Refresh Grid).  They only need parameters/configurations to work and deliver business logic. Usually, actions need some input parameters based on which they will execute their specific code. Some of the actions also have output tokens, resulting from the code being executed, that can be used in the next actions. This will give you the ability to create complex flows and obtain your custom business logic. When the workflow ends, you can attach the output tokens from the actions to the workflow's output parameters.
```